### PR TITLE
Disable font-size for the Featured Category block #5681

### LINKED ISSUE #5681: Featured Category: Font-Size Global Styles implementation has changed the default design
## Describe the bug
The implementation of the `font-size` with the Global Styles API has changed the default design of the `Featured Category` block.

## To reproduce

1. Be sure that you have activated a block theme and you have WordPress 5.9 or Gutenberg enabled.
2. Open the editor and add the `Feature Category` Block.
3. Check that description and title have the same font size. (in the default design, the font size of the title is bigger than the description).

## Expected behavior
The font size of the title should be bigger than the description.

## Screenshots

image
*current and wrong design*

image
*correct design*

### PR DESCRIPTION
**Pull request by gigitux**
The implementation of the font size with the Global Styles API has changed the default design of the Featured Category block.

This PR disables the possibility to change the font size with the Global Styles API and restore the old and correct default design.

Fixes #5681 

### Screenshots

image
*Featured Category Block using the TT1 Block Theme with the correct default design*
### Manual Testing

How to test the changes in this Pull Request:

Check out this branch

1. Be sure to activate a block theme and have WordPress 5.9 or WordPress 5.8 with Gutenberg enabled.
2. Open the editor and add the `Feature Category` Block.
3. Check that description and title have different font size and look equal to the attached image.

### CODE REVIEW

File: assets/js/blocks/featured-category/style.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .wp-block-woocommerce-featured-category {
-	background-color: #000;
+	background-color: $gray-900;
```
Comment (Aljullu) [inline assets/js/blocks/featured-category/style.scss:2]: Just to confirm, this change is unrelated to this fix, right? You are setting the background color to [how it was](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5542/files#diff-b33081124123f537259e21c9344567df96ba2ecf59e2cbc6ff4da1ec61aeb9e1L3) before #5542, correct?

LGTM but just wanted to make sure I'm understanding it. :smile:

**Review by Aljullu — APPROVED** [review]
This is testing well and code looks good. Just left one question below. :point_down:

File: assets/js/blocks/featured-category/style.scss (hunk 1 above)
Comment (gigitux) [reply assets/js/blocks/featured-category/style.scss:2]: Yes, I noticed the difference and I fixed it! 👍
